src/utils/constants.js

```diff
@@ -8,6 +8,11 @@ export const emailRegex =
 
 export const mobileRegex = /^[0-9]{10}$/;
 
+export const otpRegex = /^[0-9]{6}$/;
+
+export const authLeftPanelImage =
+  "https://image.shutterstock.com/z/stock-vector-set-of-fruit-and-vegetables-logo-for-groceries-agriculture-stores-packaging-and-advertising-318421853.jpg";
```

Comment:
remember that this will be removed after we use the image from assets

Reply:
ok done